torch/_inductor/graph.py

```diff
@@ -227,6 +224,21 @@ def decide_layout_opt(gm) -> bool:
         if nconv == 0:
             return False
 
+        # For cpu backend and mkldnn enabled, we always using channels_last for a better performance.
+        if (
+            all(
+                n.args[idx].meta["val"].device == torch.device("cpu")
+                for n in conv_nodes
+                for idx in [0, 1]
+            )
+            and torch.backends.mkldnn.enabled
+            and torch.backends.mkldnn.is_available()
+        ):
+            return True
+
+        if not config.layout_optimization:
```

Comment:
Can we move this back to the beginning of this method? This can be used as a kill switch (it's on by default). It would be weird if one disable layout_optimization (for whatever testing reason) and found it's still enabled due to cpu tensors.

Reply:
Yes, updated.